test/core/credentials/transport/alts/alts_security_connector_test.cc

```diff
@@ -132,7 +133,7 @@ TEST(AltsSecurityConnectorTest, UnknownPeerPropertyFailure) {
   tsi_peer_destruct(&peer);
 }
 
-static bool test_identity(const grpc_auth_context* ctx,
+static bool test_property(const grpc_auth_context* ctx,
```

Comment:
For the functions that we change, can we please C++-ify them? (I mention this elsewhere, but these files were originally written in C98 and we've been C++-ifying them as we go.)

In this case, perhaps we can change the function signature to

```
bool IsPeerAuthenticatedWithPeerProperty(const grpc_auth_context* ctx, absl::string_view property_name, absl::string_view property_value)
```

Ideally we would also move this function into an anonymous namespace at the top of the file rather than marking it static, but this is optional.